src/App.js

```diff
@@ -59,7 +60,15 @@ class App extends Component {
       })
     }
 
+    deleteCard = (deletedCard) => {
+      //set the state of the cards so the array of display cards no longer contains that country
+      this.setState({
+        displayCards: this.state.displayCards.filter(item => item !== deletedCard.props.country)
+        })
+    }
+
```

Comment:
Great use of the filter method to trigger a state update of the App component, by using the setState() method.